.github/workflows/stale.yaml

```diff
@@ -0,0 +1,29 @@
+name: Close Stale
```

Comment:
Minor thing, but can we call this just "Stale" since we're not actually closing stuff (at least not yet)

Reply:
Of course!